application/config.json.template

```diff
@@ -1,6 +1,6 @@
 {
     "token": "<put_your_token_here>",
-    "gistApiKey": "<your_gist_personal_access_token>",
+    "githubApiKey": "<your_github_personal_access_token>",
```

Comment:
which permission checkboxes have to be clicked on the token now? before, it was just `gist`. and now?

Reply:
I selected public repo. not sure if it would even work without it

Reply:
can u put that info in the PR description please? im going through them when i do releases, to tell people how to migrate

Reply:
> not sure if it would even work without it

test it then